specification/network/resource-manager/readme.md

```diff
@@ -410,6 +410,96 @@ input-file:
 - Microsoft.Network/preview/2015-05-01-preview/network.json
 ```
 
+## Suppression  
+``` yaml

```

Comment:
@MikhailTryakhov  For conditional suppressions, you simply add conditions like so
```
yaml $(tag) == 'package-2015-06 || $(tag) == 'package-2015-05-preview` 
```
and so on until all the tags are included in the condition.
@fearthecowboy does that sound right?

Reply:
@dsgouda  is this to restrict version where we apply this exception? 
as I understood @ravbhatnagar approved this to apply on all the versions cause we must change it in older version and don't plan to change in future (huge breaking change)